stubs/django/urls/conf.pyi

```diff
@@ -0,0 +1,8 @@
+from typing import Any, Mapping, Optional, Union
+
+from django.urls.resolvers import URLPattern, URLResolver
+
+def include(arg: Any, namespace: Optional[str] = ...) -> Tuple[Any, str, str]: ...
```

Comment:
You'll need a `# pyre-unsafe` or `# pyre-ignore` to make pyre OK with the `Any`s.